docs/source/teams/roles_and_permissions.rst

```diff
@@ -105,14 +139,28 @@ Admins and users with the **Can manage** permission on a dataset can configure
 a dataset's permissions under the dataset's
 :ref:`Manage tab <teams-managing-datasets>` in the FiftyOne Teams App.
 
-A user's permissions on a dataset is the maximum of their permissions from the
+In FiftyOne Teams, dataset permissions for a user are determined by both the 
+access they receive from their group's permissions and individual permissions 
```

Comment:
nit: change `group's` to `group(s)` to convey that a user may be a member of multiple groups